tagging_accuracy_report.py

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/env python3
+import boto3
+import botocore
+from botocore.exceptions import ClientError
+from nltk.stem import PorterStemmer
+import json
+import os
+import sys
+
+
+class tagging_accuracy_report():
+
+    def __init__(self,
+                 manual_tags_bucket,
+                 automated_tags_bucket,
+                 ecosystem='maven'):
+        self.manual_tags_bucket_name = manual_tags_bucket
+        self.automated_tags_bucket_name = automated_tags_bucket
+        aws_access_key_id = os.environ.get("AWS_ACCESS_KEY_ID", None)
+        aws_secret_access_key = os.environ.get("AWS_SECRET_ACCESS_KEY", None)
+        if aws_access_key_id is None or aws_secret_access_key is None:
+            print("No AWS credentials set in environment")
+            sys.exit(1)
+        self.session = boto3.session.Session(
+            aws_access_key_id, aws_secret_access_key)
+        self.s3_resource = self.session.resource(
+            's3', config=botocore.client.Config(signature_version='s3v4'))
+        self.stemmer = PorterStemmer()
+        self.ecosystem = ecosystem
+
+    def load_manual_tags_json(self, manual_tags_filename):
+        manual_tags_file = self.s3_resource.Object(
+            self.manual_tags_bucket_name, manual_tags_filename
+        ).get()['Body'].read().decode('utf-8')
+        manual_tags_json = json.loads(manual_tags_file)
+        for tag_data in manual_tags_json:
+            if tag_data['ecosystem'] == self.ecosystem:
+                self.package_topic_json = tag_data['package_topic_map']
+                break
+
+    def get_automated_tags_for_package(self, package_name):
+        tags_file_path = "{}/{}.json".format(self.ecosystem, package_name)
+        automated_tags_file = self.s3_resource.Object(
+            self.automated_tags_bucket_name, tags_file_path
+        ).get()['Body'].read().decode('utf-8')
+        automated_tags_json = json.loads(automated_tags_file)
+        return automated_tags_json
+
+    def match_tags(self):
+        no_tags = man_tags_not_in_result = man_tags_not_collected = 0
+        for package_name in self.package_topic_json:
+            # print("Package: " + package_name)
+            manual_tags = set(self.package_topic_json[package_name])
+            # print(manual_tags)
+            try:
```

Comment:
I suggest moving this logic to `get_automated_tags_for_package` or just don't use `get_automated_tags_for_package` since it just wraps library calls. 

I would prefer to `get_automated_tags_for_package` return set of tags (empty set if none are found).